plugins/trace_api_plugin/store_provider.cpp

```diff
@@ -206,6 +250,25 @@ namespace eosio::trace_api {
       }
    }
 
+   bool slice_directory::find_or_create_trx_id_slice(uint32_t slice_number, open_state state, fc::cfile& trx_id_file) const {
+       const bool found = find_trx_id_slice(slice_number, state, trx_id_file);
+       if( !found ) {
+           trx_id_file.open(fc::cfile::create_or_update_rw_mode);
+       }
+       return found;
+   }
+
+   bool slice_directory::find_trx_id_slice(uint32_t slice_number, open_state state, fc::cfile& trx_id_file, bool open_file) const {
+      const bool found = find_slice(_trace_trx_id_prefix, slice_number, trx_id_file, open_file);
+      if( !found || !open_file ) {
+         return found;
+      }
+      if( state == open_state::write ) {
+          trx_id_file.seek_end(0);
+      }
```

Comment:
Do we need to `seek(0)` like in `find_trace_slice` ?

Reply:
`seek(0)` is called inside of `find_slice()`, which is called on line#262.